softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java

```diff
@@ -9,7 +9,7 @@
  */
 
 @Component
-public class HandlerFactory<T extends Handler> extends AbstractFactory<T> {
+public class HandlerFactory<T extends RequestHandler> extends AbstractFactory<T> {
```

Comment:
@AlexKazmirchuk Why RequestHandler? Factory can work with handlers and with routers